src/modules/ListItemMod.js

```diff
@@ -0,0 +1,127 @@
+import {
+  createElementDefault, createImg, createButton, createNS, createLabel,
+  createTextArea, createInput,
+} from './createElementMod.js';
+import { countTotalItem } from './CountTotalItems.js';
+
+const toggleBlur = () => {
+  const body = document.querySelector('body');
+  body.classList.toggle('modal-active');
+};
+
+const closePop = () => {
+  toggleBlur();
+  const overlay = document.querySelector('.app-item-popup-overlay');
+  overlay.classList.add('out');
+  const popup = document.getElementById('popup');
+  setTimeout(() => popup.removeChild(overlay), 400);
+};
+
+const createItem = (elem, classes, mealApi, xlink, likeApi,
+  callback0 = false, callback1 = false) => {
+  const docFrag = document.createDocumentFragment();
+  docFrag.appendChild(createImg(classes.image, mealApi.strMeal, mealApi.strMealThumb));
+  const elemHeader = createElementDefault('div', classes.titleHeader);
+  const titleImg = createElementDefault('p', classes.likesCtn, mealApi.strMeal);
+  elemHeader.appendChild(titleImg);
+  const elemLikes = createElementDefault('div', classes.likesIcon);
+  const elemLikesSVG = createNS(xlink.regular);
+  const elemLikesRedSVG = createNS(xlink.red);
+  elemLikesRedSVG.classList.add('hidden');
+  const elemLikesP = createElementDefault('p', classes.likesText, `${likeApi} like${likeApi > 1 ? 's' : ''}`);
+  elemLikes.append(elemLikesSVG, elemLikesRedSVG, elemLikesP);
+  elemHeader.appendChild(elemLikes);
+  docFrag.appendChild(elemHeader);
+  const btnDiv = createElementDefault('div', classes.btnCtn);
+  const commentBtn = createButton('button', classes.btnComment, 'Go to Comments', 'Comments');
+  if (callback1) commentBtn.addEventListener('click', callback1);
+  btnDiv.appendChild(commentBtn);
+  btnDiv.appendChild(createButton('button', classes.btnReserve, 'Go to Reservations', 'Reservations'));
```

Comment:
-[ ] 🙏Please remove the reservation button since it's not functioning. This will ensure that your project only includes the parts that are working👍, resulting in a cleaner🗑 implementation and a better user👩‍🦰 experience. 
![image](https://github.com/lordksix/Meals-API-template/assets/63445083/700fcedc-f729-4a26-9eb8-a214e4953fb7)

Reply:
Thank you for the feedback. It was removed in commit 2dea44b72e5e45df6ebb67215aafa3c961fb77de